src/components/ha-form/ha-form-integer.ts

```diff
@@ -39,20 +44,31 @@ export class HaFormInteger extends LitElement implements HaFormElement {
       ? html`
           <div>
             ${this.label}
-            <ha-paper-slider
-              pin=""
-              .value=${this._value}
-              .min=${this.schema.valueMin}
-              .max=${this.schema.valueMax}
-              @value-changed=${this._valueChanged}
-            ></ha-paper-slider>
+            <div class="flex">
+              ${this.schema.optional
```

Comment:
optional and no default value. If there is a default value, we should just render that as that's the value that will go into the system.

Reply:
But it is optional? So you should be allowed to not set a value?

Reply:
But if you don't give a value, we will fill in a value. So we should not give the illusion that you can opt-out of giving a value.

Reply:
Ah ok 👍